recommendation_v2/torchrec_dlrm/dlrm_main.py

```diff
@@ -551,32 +691,38 @@ def main(argv: List[str]) -> None:
     else:
         device: torch.device = torch.device("cpu")
         backend = "gloo"
+    dist.init_process_group(backend=backend)
 
-    if rank == 0:
-        print(
-            "PARAMS: (lr, batch_size, warmup_steps, decay_start, decay_steps): "
-            f"{(args.learning_rate, args.batch_size, args.lr_warmup_steps, args.lr_decay_start, args.lr_decay_steps)}"
+    is_rank_zero = dist.get_rank() == 0
+    if is_rank_zero:
+        pprint(vars(args))
+        submission_info(mllogger, "dcnv2", "reference_implementation")
+        mllogger.event(
+            key=mllog_constants.GLOBAL_BATCH_SIZE,
+            value=dist.get_world_size() * args.batch_size,
+        )
+        mllogger.event(
+            key=mllog_constants.OPT_BASE_LR,
+            value=args.learning_rate,
+        )
+        mllogger.event(
+            key=mllog_constants.GRADIENT_ACCUMULATION_STEPS,
+            value=1,  # Gradient accumulation is not supported in the reference implementation
```

Comment:
It should log the optimizer name somewhere around here, or even better - close to line 822

Reply:
done here https://github.com/mlcommons/training/blob/bc163f0540575adf4a73c056d870f16daef89a3f/recommendation_v2/torchrec_dlrm/dlrm_main.py#L849-L852